### ISSUE TITLE
Deep learning of quantitative MRI parameters

### ISSUE THREAD

**Issue by csparker** [CMIC-UCL/CMICHACKS#7]
**Project leader:** Chris Parker (christopher.parker@ucl.ac.uk)

### Description:
qMRI done better. The aim is to update quantitative MRI machine learning algorithms so that they correctly handle noise in MRI images.

### Participant requirements:
One of: Python, PyTorch, Machine Learning, Quantitative MRI, Diffusion MRI, Modelling

### Resources:
We will mainly use simulated data based on diffusion MRI models (example [here](https://pubs.rsna.org/doi/10.1148/radiology.161.2.3763909)). We will improve [current methods](https://arxiv.org/abs/2205.05587) for unsupervised learning and update an [existing GitHub repository](https://github.com/sebbarb/deep_ivim/blob/master/deep_ivim_demo.ipynb) with our new code. For example a walk-through and model evaluation will be added to that to show the improvement gained by using our technique. 

### First Issues:
- [ ] Make a new PyTorch-compatable loss function based on the Rician likelihood.
- [ ] Evaluate performance of the parameter estimation.
- [ ] Publish as GitHub repository.
- [ ] Extend to bayesian framework / test on other dMRI models / demonstrate with real data.
